package.json

```diff
@@ -140,11 +140,12 @@
     "istanbul": "latest",
     "mocha": "latest",
     "nock": "2.17.0",
+    "nodemon": "^1.11.0",
```

Comment:
Woops! It turns out there's a good reason for having the caret range, which is the possibility of updating to `v2.x` and the config file no longer works (`v2.x` might not be backwards compatible). So I say we keep it as is because it's safer this way.